src/net_processing.cpp

```diff
@@ -3224,6 +3224,10 @@ void PeerManagerImpl::ProcessMessage(CNode& pfrom, const std::string& msg_type,
         // Change version
         const int greatest_common_version = std::min(nVersion, PROTOCOL_VERSION);
         pfrom.SetCommonVersion(greatest_common_version);
+        {
```

Comment:
we really shouldn't have this change being performed from this repo :)

Reply:
ping @luke-jr

Reply:
@hebasto ping seen. nothing to do here though.

Reply:
> ping seen. nothing to do here though.

This PR modifies `src/net_processing.cpp`, and we review such changes in the main repo, generally.

Mind moving this PR or the first commit from this repo into the main one?

Reply:
Doesn't seem worth splitting up such a minor change IMO